www/privacy/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page 
+title: Privacy Policy 
+---
+
+# Privacy Policy
+
+When you opt into telemetry, information about your usage of Apache Cordova CLI is collected. The collected information consists of the following:
+
+* IP address
+* OS type and version
+* Node version
+* Cordova version
+* Commands executed, time of execution and status (success/failure)
+
+This information is saved in Google Analytics and handled by Google as described in their [privacy policy](http://www.google.com/policies/privacy/). See the documentation for instructions on how to disable telemetry if you prefer not to share your data.
+
+We use the gathered information to help us make our tool more useful and to better understand how it is used. We do not track or collect personally identifiable information or associate gathered data with any personally identifying information from other sources.
+
+By opting into telemetry, you consent to the collection of this data in the manner and for the purpose described above.
+
+Apache Cordova welcomes your questions or comments regarding this Privacy Policy. Send them to dev@cordova.apache.org
```

Comment:
The Apache Cordova __community__ welcomes your questions or comments regarding this Privacy Policy. Send them to dev@cordova.apache.org